transforms:
  T_0S:
    - [-1.0, 0.0, 0.0, 0.358]
    - [0.0, 1.0, 0.0, 0.03]
    - [0.0, 0.0, -1.0, 0.006]
    - [0.0, 0.0, 0.0, 1.0]
  
  #camera 33137761
  H1:
    - [0.9569, 0.132, -0.2586, 0.2624]
    - [-0.29, 0.4784, -0.8289, 0.6944]
    - [0.0143, 0.8682, 0.4961, -0.3523]
    - [0.0, 0.0, 0.0, 1.0]

  # camera 36829049
  H2:
    - [-0.9997, -0.0225, 0.001, 0.1255]
    - [0.0144, -0.6029, 0.7977, -0.8284]
    - [-0.0174, 0.7975, 0.6031, -0.582]
    - [0.0, 0.0, 0.0, 1.0]


